gem/lib/pagy/backend.rb

```diff
@@ -3,7 +3,7 @@
 
 class Pagy
   # Define a few generic methods to paginate a collection out of the box,
-  # or any collection by overriding pagy_get_items and/or pagy_get_vars in your controller
+  # or any collection by overriding pagy_get_items and/or pagy_get_count in your controller
```

Comment:
Nicely spotted!

```diff
-  pagy_get_vars
+  pagy_get_count
```

Any of the `pagy_get_` methods may be overridden (for whatever reason): should we alert users to this possibility rather than confine the patch to just `pagy_get_items` and `pagy_get_count`?

Reply:
> Any of the `pagy_get_*` methods...

👍🏼 @benkoshy since it's a code comment... yes, a general comment may be more appropriate.

We should consider also adapting it to the backend extras.

Reply:
@benkoshy Ah yes fair enough. I wasn't sure so I went for the least changes.     
Ive updated the comment to:
```ruby
# Define a few generic methods to paginate a collection out of the box,
# or any collection by overriding any of the `pagy_get_` methods in your controller.
# See also the extras if you need specialized methods to paginate Arrays or other collections
```

Do you think that's better? Or do you have some better suggestions?

Reply:
> Do you think that's better? Or do you have some better suggestions?

Looks great!

BTW: if you want bonus marks - consider adapting a similar comment in the "extras". i.e. to alert to the possibility that those methods can be overridden.  Or I can do it for you and credit you with the PR.

Reply:
> consider adapting a similar comment in the "extras".

Hm, are you talking about the [Pagy::PagyExtra](https://github.com/ddnexus/pagy/blob/master/gem/lib/pagy/extras/pagy.rb)?    
I've never overridden those and I'm not sure if I can provide a clear comment instruction on how to override them.    
Seems like you need make some [extra steps to override frontend methods.](https://benkoshy.github.io/2020/02/01/overriding-pagy-methods.html#user-content-b-pagy-front-end-methods)

I would be happy if someone else with actual experience can provide a comment there.

Reply:
> Hm, are you talking about the [Pagy::PagyExtra](https://github.com/ddnexus/pagy/blob/master/gem/lib/pagy/extras/pagy.rb)?

- Pagy has some "[extras](https://github.com/ddnexus/pagy/tree/master/gem/lib/pagy/extras)" with methods that be overriden. A comment similar to what you suggested above would suffice?

Reply:
I have updated with a note suggesting that backend methods can be overridden and have credited Leon Vogt

if you are satisfied i can squate / rebase on master?

Reply:
here is the full branch with missing comments (sorry I forgot to paste it): https://github.com/ddnexus/pagy/commits/update-backend-comment/ 

- https://github.com/ddnexus/pagy/commit/81b781621fe7b2cd3288e242266fba8448c30856
- https://github.com/ddnexus/pagy/commit/3fbf56be68f98b8a31049b869ecb186cf4a1e402

Perhaps cherry pick those comments, remove my name, add yours, and you'll be winning.

Reply:
Awesome thank you! I've cherry picked the two commits. And I will for sure not remove your name! You have done all the work! 😂
Do you want me to squash them or are you doing that while merging?